index.html

```diff
@@ -152,32 +202,46 @@
 The <dfn id="widl-Navigator-connection">`connection`</dfn> attribute, when getting, returns an object that implements the <a>`NetworkInformation`</a> interface.
 </section>
 
-<section>
+<section data-dfn-for="NetworkInformation" data-link-for="NetworkInformation">
 ## The `NetworkInformation` interface
 The <dfn>`NetworkInformation`</dfn> interface provides a means to access information about the network connection the user agent is currently using.
 
 <pre class="idl">
 [Exposed=(Window,Worker)]
 interface NetworkInformation : EventTarget{
   readonly attribute ConnectionType type;
+  readonly attribute EffectiveConnectionType effectiveType;
   readonly attribute Megabit downlinkMax;
+  readonly attribute Megabit downlink;
+  readonly attribute Millisecond rtt;
   attribute EventHandler onchange;
 };
 
 typedef unrestricted double Megabit;
+typedef unsigned long long Millisecond;
```

Comment:
@igrigorik make sure this matches whatever is used in other specs that rely on millis.